mobile/Screens/PocketPromptScreen.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import {
+  StyleSheet, Text, View, TouchableOpacity, Dimensions,
+} from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import axios from 'axios';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#151716',
+  },
+  message: {
+    fontSize: 24,
+    paddingHorizontal: 25,
+    marginBottom: 180,
+    color: '#FFFFFF',
+  },
+  reshuffleButton: {
+    backgroundColor: '#84C2C9',
+    marginTop: Dimensions.get('window').height / 2.2,
```

Comment:
Dividing by 1.5 instead seemed to work better on my device! But feel free to test and keep your original if it looks better on your end!!
`    marginTop: Dimensions.get('window').height / 1.5,`